net-misc/stargazer/metadata.xml

```diff
@@ -1,54 +1,52 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Change this back to double quotes. While single quotes are OK, double quotes are canonical.

Reply:
Done.